grafana/provisioning/dashboards/qumulo/cluster.json

```diff
@@ -2051,6 +2051,6 @@
   "timezone": "",
   "title": "Cluster Overview",
   "uid": "isJoSTGVz",
-  "version": 6,
+  "version": 2,
```

Comment:
IIUC, version is just metadata to look between different changes to a dashboard? Why are we going backwards/pegging to 2 and why do both of these dashboards have the same value?